sections/main-order.liquid

```diff
@@ -299,10 +299,25 @@
               </td>
             </tr>
           {%- endif -%}
+          {%- if order.total_refunded_amount > 0 -%}
+            <tr role="row">
+              <td id="RowTotalRefund" role="rowheader" scope="row" colspan="3">
+                {{ 'customer.order.total_refunded' | t }}
+              </td>
+              <td
+                headers="RowTotalRefund"
+                role="cell"
+                colspan="2"
+                data-label="{{ 'customer.order.total_refunded' | t }}"
+              >
+                {{ order.total_refunded_amount | money_with_currency }}
```

Comment:
I was wondering if the math worked well when you partially refund shipping on top of that and it worked well but the only thing is that we don't display the subsequent refunds. Unsure if it's relevant to buyers to be aware if there was multiple attempt to refund partially an order. 🤔 

- This is my test order `#1041`  [admin](https://screenshot.click/19-39-6qxr3-mqtjt.png)
- This is the [storefront account](https://screenshot.click/19-42-0lghy-ywcob.png)

Reply:
I feel like maybe it's added noise, I'm not sure how useful it is to be displaying each refund individually 🤔 Plus if we do that then it means we will also need a total which will add even more info/content to the order. 

On the [order object](https://shopify.dev/api/liquid/objects/order), there are the transactions information available but it doesn't contain much info. We can check if a transaction is a `refund` and output its amount, but again not sure how useful. Here is how it could look: 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/20-12-02ge6-okc6i.png)
</details>

Here is the information available for transactions: 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/20-13-nth71-0lnml.png)
</details>

I think if we could have had more details about what was refunded in that transaction it'd be more useful but right now it doesn't seem to add much. The total refund amount seem enough.